argument_example.rb

```diff
@@ -0,0 +1 @@
+
```

Comment:
You might want to remove this file. Seems like it was mistakenly added.

Reply:
Looks like it, I will remove 👍